File: async-scala-client/src/main/scala/io/swagger/client/model/IpbtConfItem.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class IpbtConfItem (
  state: Option[StateEnum],
federalTax: Option[Double],
importTax: Option[Double],
stateTax: Option[Double],
cityTax: Option[Double],
source: Option[String]
)

File: javascript-closure-angular-client/API/Client/IpbtConfItem.js
goog.provide('API.Client.IpbtConfItem');

/**
 * @record
 */
API.Client.IpbtConfItem = function() {}

/**
 * @type {!API.Client.StateEnum}
 * @export
 */
API.Client.IpbtConfItem.prototype.state;

/**
 * @type {!number}
 * @export
 */
API.Client.IpbtConfItem.prototype.federalTax;

/**
 * @type {!number}
 * @export
 */
API.Client.IpbtConfItem.prototype.importTax;

/**
 * @type {!number}
 * @export
 */
API.Client.IpbtConfItem.prototype.stateTax;

/**
 * @type {!number}
 * @export
 */
API.Client.IpbtConfItem.prototype.cityTax;

/**
 * @type {!string}
 * @export
 */
API.Client.IpbtConfItem.prototype.source;

